db/query.go

```diff
@@ -38,7 +38,7 @@ func ExecuteQuery(queryString string) (*results.ResultStreamer, error) {
 	utils.FailOnErrorWithMessage(err, "client failed to initialize")
 
 	// refresh connections
-	connectionsUpdated, err := client.RefreshConnections()
+	_, err = client.RefreshConnections()
```

Comment:
update RefreshConnections to no longer return the bool - this is the only place it was used